Application/server/tests/apiContainer.test.js

```diff
@@ -0,0 +1,78 @@
+const request = require('supertest');
+const async = require('async');
+
+describe('POST /containerDetails', () => {
+	var containersId
+	beforeAll(async () => {
+		const response = await request('http://localhost:8080')
+			.get('/api/container/listall')
+			.set('Content-Type', 'application/json')
+			.set('Accept', 'application/json')
+			.set('Authorization', 'defaultToken')
```

Comment:
Change all "defaultToken" to a valid token. You can get some hints by looking the other tests